regolith/stylers.py

```diff
@@ -2,6 +2,12 @@
 
 import re
 
+month_fullnames = {1: 'January', 2: 'February', 3: 'March', 4: 'April', 5: 'May',
```

Comment:
Can these go with the other month translation maps in https://github.com/regro/regolith/blob/master/regolith/dates.py ?